.github/workflows/test-aks.yaml

```diff
@@ -0,0 +1,363 @@
+      - name: Build images
+        if: steps.gen-id.outputs.RUN_TEST == 'true' && matrix.images != ''
+        run: |
+          # split images and directories into arrays
+          IFS=',' read -ra images <<< "${{ matrix.images }}"
+          IFS=',' read -ra directories <<< "${{ matrix.directories }}"
+          echo "Building images: $images"
+          echo "Directories: $directories"
+          for index in "${!images[@]}"; do
+            image=${images[$index]}
+            directory=${directories[$index]}
+            echo "Building image $image from directory $directory"
+            docker build -t ghcr.io/radius-project/$image:${{ github.run_id }} $directory
+            docker tag ghcr.io/radius-project/$image:${{ github.run_id }} ghcr.io/radius-project/$image:${{ github.run_id }}
+            # We need to think about how we can delete these images at the end of the run
+            docker push ghcr.io/radius-project/$image:${{ github.run_id }}
+          done
+      # Create and install test environment
+      - name: Create Azure resource group
+        if: steps.gen-id.outputs.RUN_TEST == 'true' && matrix.credential == 'azure'
+        id: create-azure-resource-group
+        run: |
+          current_time=$(date +%s)
+          az group create \
+            --location ${{ env.AZURE_LOCATION }} \
+            --name ${{ steps.gen-id.outputs.TEST_AZURE_RESOURCE_GROUP }} \
+            --subscription ${{ secrets.AZURE_SUBSCRIPTIONID_TESTS }} \
+            --tags creationTime=$current_time
+          while [ $(az group exists --name ${{ steps.gen-id.outputs.TEST_AZURE_RESOURCE_GROUP }} --subscription ${{ secrets.AZURE_SUBSCRIPTIONID_TESTS }}) = false ]; do
+            echo "Waiting for resource group ${{ steps.gen-id.outputs.TEST_AZURE_RESOURCE_GROUP }} to be created..."
+            sleep 5
+          done
+      - name: Create EKS Cluster
+        if: steps.gen-id.outputs.RUN_TEST == 'true' && matrix.credential == 'aws'
+        id: create-eks
+        run: |
+          curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
+          sudo mv /tmp/eksctl /usr/local/bin
+          eksctl create cluster \
+            --name ${{ steps.gen-id.outputs.TEST_EKS_CLUSTER_NAME }} \
+            --nodes-min 1 --nodes-max 2 --node-type t3.large \
+            --zones ${{ env.AWS_ZONES }} \
+            --managed \
+            --region ${{ env.AWS_REGION }}
+          while [[ "$(eksctl get cluster ${{ steps.gen-id.outputs.TEST_EKS_CLUSTER_NAME }} --region ${{ env.AWS_REGION }} -o json | jq -r .[0].Status)" != "ACTIVE" ]]; do
+            echo "Waiting for EKS cluster to be created..."
+            sleep 60
+          done
+          aws eks update-kubeconfig --region ${{ env.AWS_REGION }} --name ${{ steps.gen-id.outputs.TEST_EKS_CLUSTER_NAME }}
+        timeout-minutes: 60
+        continue-on-error: false
+      - name: Install Dapr
```

Comment:
If you use /test/azure/ bicep template, dapr is installed and maintained by AKS. So I would disable this one for AKS cluster.